transport_test.go

```diff
@@ -173,3 +176,72 @@ func TestNew_appendHeader(t *testing.T) {
 		t.Errorf("could not find %v in request's accept headers: %v", myheader, headers["Accept"])
 	}
 }
+
+func TestRefreshTokenWithParameters(t *testing.T) {
+	installationTokenOptions := &github.InstallationTokenOptions{
+		RepositoryIDs: []int64{1234},
+		Permissions: &github.InstallationPermissions{
+			Contents: String("write"),
+			Issues:   String("read"),
```

Comment:
Have I missed something, or where is the `String` function defined? Are you referring to `github.String()`?